staff/marc-salas/precurse/CalculadoraHTML/calculadoraHTML.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="calculadoraHTML.css">
+</head>
+<body>
+    <script src="calculadoraHTML.js"></script>        
+    <div class="container">
+        <div class="resultado" id="actual"></div>
+        <div class="resultado" id="resultado"></div>
+        <div class="tecla" onclick="numero(this)" id="1">1</div>
+        <div class="tecla" onclick="numero(this)" id="2">2</div>
+        <div class="tecla" onclick="numero(this)" id="3">3</div>
+        <div class="tecla" onclick="operacio(this)" id="+">+</div>
+        <div class="tecla" onclick="numero(this)" id="4">4</div>
+        <div class="tecla" onclick="numero(this)" id="5">5</div>
+        <div class="tecla" onclick="numero(this)" id="6">6</div>
+        <div class="tecla" onclick="operacio(this)" id="-">-</div>
+        <div class="tecla" onclick="numero(this)" id="7">7</div>
+        <div class="tecla" onclick="numero(this)" id="8">8</div>
+        <div class="tecla" onclick="numero(this)" id="9">9</div>
+        <div class="tecla" onclick="operacio(this)" id="*">*</div>
+        <div class="tecla" onclick="operacio(this)" id=".">.</div>
+        <div class="tecla" onclick="numero(this)" id="0">0</div>
+        <div class="tecla" onclick="ac(this)" id="A">AC</div>
+        <div class="tecla" onclick="operacio(this)" id="/">/</div>
+        <div class="igual" onclick="igual(this)"" id="=">=</div>
```

Comment:
Hey Marc, nice project but remember this when you make a button :
 
From a usability perspective, it's better to implement <button> over <div> or <a> due to the way browsers handle interactivity.
If your users are sloppy clickers and they click-and-drag by mistake, or on a touch-screen, they can end up selecting the <div> contents rather than triggering a click event. Using <button> will greatly reduce this problem.

Reply:
Thanks Enric, i will remember these for the next projects!